### ISSUE TITLE
search only nsfw extensions filter for search

### ISSUE THREAD

**Issue by developer861** [mihonapp/mihon#1300]
### Describe your suggested feature

add a filter so when searching for a manga you can chose to search only in nsfw extensions

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
Remember that a lot of extensions are usually marked as NSFW because they offer _some_ NSFW content. For those, your filter would be a lot less useful, assuming you're after NSFW content only with this filter.
Just use the normal search, maybe pin your favourite NSFW source(s), and check through the results there.

**Comment by developer861** [comment]
> Remember that a lot of extensions are usually marked as NSFW because they offer _some_ NSFW content. For those, your filter would be a lot less useful, assuming you're after NSFW content only with this filter. Just use the normal search, maybe pin your favourite NSFW source(s), and check through the results there.

yep i think you're right but the problem is that i have too many extensions installed on my phone so it can help when I'm looking for nsfw content.

i think adding an option for categorizing the extensions in global search can be a good solution too
